content/references/moderator-features.md

```diff
@@ -0,0 +1,23 @@
+---
+kind: reference
+sidebar: community
+prev: /community/rules/
+---
+
+# Moderator Features
+
+[Moderators](/community/moderation/#moderators) are a group of users selected to support users with handling problems related to content and community. They have access to tools and features not available to regular users, which can be used to solve problems reported by community members. In particular, moderators can:
+
+- Handle problematic users, cheaters, and griefers:
```

Comment:
Wow, I just learned a new word "griefers" :-D (no action required)

Reply:
Maybe it's too unofficial, or something?

Reply:
IDK, but it's a common word in Minecraft and Counter-Strike.